src/runtime_src/core/pcie/common/aie_stubs.cpp

```diff
@@ -0,0 +1,144 @@
+#include <sstream>
+
+#include "core/include/experimental/xrt_aie.h"
+#include "core/include/experimental/xrt_bo.h"
+#include "core/include/xcl_graph.h"
+
+#include "core/include/experimental/xrt_device.h"
+
+//////////// START OF XCL XRT AIE APIS ///////////////////////////
+
+static std::string function_not_implemented(const char* funcName) {
+  std::stringstream excepStreamMsg;
+  excepStreamMsg << funcName << " is not supported for hw and hw_emu PCIe targets";
+  std::string exceptionStr(excepStreamMsg.str());
+  return exceptionStr;
+}
+
+void*
+xclGraphOpen(xclDeviceHandle /*handle*/, const uuid_t /*xclbin_uuid*/, const char* /*graph*/, xrt::graph::access_mode /*am*/)
```

Comment:
Thanks Soren. I don't have windows setup locally to build PCIe XRT. Now I have the setup to run the windows build. Corrected this and ran the XRT build in the local windows machine, I could see 0 Errors and 0 Warnings now. So I assume it will be resolved now.